src/sentry/integrations/github/client.py

```diff
@@ -112,6 +113,10 @@ def create_token(self):
             },
         )
 
+    @transaction_start("GitHubClientMixin")
```

Comment:
@ceorourke the transaction should be something like: `GitHubClientMixin.get_file_url`.